Documentation/howto/revert-branch-rebase.txt

```diff
@@ -27,8 +27,8 @@ $ git checkout -b revert-c99 master
 Now I am on the 'revert-c99' branch.  Let's figure out which commit to
 revert.  I happen to know that the top of the 'master' branch is a
 merge, and its second parent (i.e. foreign commit I merged from) has
-the change I would want to undo.  Further I happen to know that that
-merge introduced 5 commits or so:
+the change I would want to undo.  Further I happen to know that there're
+5 commits or so introduced by that merge:
```

Comment:
That's not a word duplication.

Reply:
As you can see from the header in lines 1--9 of this file, this is a verbatim copy of an email by Junio Hamano. I am not all that sure that it should be considered appropriate to edit what they said, other than fixing typos and obvious style issues. This change does not strike me as an obvious typo or style improvement...

Reply:
agree